src/dockleHelper.ts

```diff
@@ -29,6 +32,25 @@ const TITLE_TITLE = "TITLE";
 const TITLE_SEVERITY = "SEVERITY";
 const TITLE_DESCRIPTION = "DESCRIPTION";
 
+export async function runDockle(): Promise<number> {
+    const docklePath = await getDockle();
+    core.debug(util.format("Dockle executable found at path ", docklePath));
+    const dockleEnv = await getDockleEnvVariables();
+    const imageName = inputHelper.imageName;
+
+    const dockleOptions: ExecOptions = {

```

Comment:
we can have small function like:
getDocklePath(..);
getDockleArguments(..);
getDokleOptions(..);
and then toorunner.exec();

Reply:
Okay. I've put all of it in a single function as most of these are single line statements anyway. But may be we can delegate these to a few functions as you suggested.